### ISSUE TITLE
Better definition what what fixes are in what nuget packages

### ISSUE THREAD

**Issue by grahambunce** [Azure/azure-webjobs-sdk#1590]
I'm trying to relate your Sprints with your nuget package releases, I want to know if a fix in Sprint 11 has been released in a package but I cannot figure out how to do this. You have a NuGet package of 2.1.0 released a month ago, but a 3.0.0-beta4 released 3 months ago.

Is Sprint 11 (for example) in 2.1.0 or in 3.0.0-beta4.... or neither. Please improve your tagging so we know what is released and what isn't

**Comment by mathewc** [comment]
Have you checked our releases page? https://github.com/Azure/azure-webjobs-sdk/releases

**Comment by grahambunce** [comment]
@mathewc ... no, because that is not available from the main page. If you go to https://github.com/Azure/azure-webjobs-sdk, which most people will, there is no icon or hyperlink or anything on that landing page to link you to the releases page.

Also, it doesn't help me as some releases appear to have links to issues, some do not. Some have detailed descriptions, others have a terse comment.

If you check the issue, you'll see that there is no way AFAIK for me to know if something fixed in Sprint 11 is actually released and if so in what package. You appear to have started to label issues with a 3.x moniker, which is a good start, but it still doesn't tell me what package I need to download to get a fixed issue.

So I don't consider this closed.

**Comment by paulbatum** [comment]
I agree, the quality of our release notes could be improved, makes sense to keep this open until that happens.

**Comment by rgamage** [comment]
I agree on the accessibility of the release notes page.  No way to discover that page from the home page, and critical info is missing.  For example, going from 3.x to 4.x sounds like a major, possibly breaking change - I see no way to tell what really happened there or why the big version jump.  Should I upgrade to 4.x?  No idea.  4.0.0 preview1 notes are missing, which is where I would look for the details on the version number jump.
